dashboard/dashboard-front/src/services/UserDataStore.ts

```diff
@@ -81,3 +83,17 @@ function saveUserData() {
         localStorage.setItem('userData.authToken', authToken.value)
     }
 }
+
+export function hasAuthCookie(): boolean {
+    return getCookie(AUTH_COOKIE_NAME) != null
```

Comment:
It's impossible for `getCookie` function to return anything else than a `null` or a `string` and the string can't be empty because of `... || null` so I would go with strict equality here.

```suggestion
    return getCookie(AUTH_COOKIE_NAME) !== null
```